app/controllers/items_controller.rb

```diff
@@ -0,0 +1,5 @@
+class ItemsController < ApplicationController
+  def index
+    @items = Item.includes(:supermarket, :customers).all
```

Comment:
Interesting. I actually really like the use of `.includes` here. When you use `.includes` you do not also need `.all`. You'll also want to be able to explain the differences in using `.includes` and `.all`